src/combobox/combobox.component.ts

```diff
@@ -58,8 +58,10 @@ import { DocumentService } from "../utils/utils.module";
 				<div
 					*ngIf="type === 'multi' && pills.length > 0"
 					(click)="clearSelected()"
+					(keydown.enter)="clearSelected()"
 					role="button"
-					class="bx--list-box__selection bx--list-box__selection--multi"
+					class="bx--list-box__selection bx--tag--filter bx--list-box__selection--multi"
```

Comment:
Do we need `bx--tag--filter` as well? I know it was working without it before

Reply:
It doesn't allow the close button to be focused without it

Reply:
Huh, odd